app/internal/calendar_privacy.py

```diff
@@ -0,0 +1,32 @@
+from app.dependencies import get_db
+from app.database.models import User
+# TODO switch to using this when the user system is merged
+# from app.internal.security.dependancies import (
+#    current_user, CurrentUser)
+
+from fastapi import Depends
+
+
+# TODO add privacy as an attribute in current user
```

Comment:
You can add it, the login is here!

Reply:
hey,
so I checked and what I need (current logged user) has yet to be merged unfortunately

https://github.com/PythonFreeCourse/calendar/pull/293/